model-optimizer/mo/utils/check_ie_bindings.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+"""
+ Copyright (C) 2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import os
+
+
+def get_mo_version():
+    version_txt = os.path.join(os.path.dirname(os.path.realpath(__file__)), os.pardir, os.pardir, "version.txt")
+    if not os.path.isfile(version_txt):
+        return "unknown version"
+    with open(version_txt) as f:
+        version = f.readline().replace('\n', '')
+    return version
```

Comment:
Is this a copy-and-paste of method from here https://github.com/openvinotoolkit/openvino/blob/master/model-optimizer/mo/utils/version.py? Why cannot we just import it from there?

Reply:
removed.